packages/geo/src/tile.cover.ts

```diff
@@ -26,4 +27,30 @@ export const TileCover = {
 
         return GeoJson.toFeatureCollection(polygons);
     },
+
+    /**
+     * Merge quadKeys if covering is > MaxPc and zoomLevel is > minZ
+     * @param c the Covering cursor - set to node to merge from
+     * @param maxPc the fraction of coverage allowd to not merge
```

Comment:
`maxPc`  doesnt really explain what this is?

`coveragePercent`?

also spelling of `allowd`